### ISSUE TITLE
Add option for transparency on message list and nav

### ISSUE THREAD

**Issue by briantully** [leggett/simplify#495]
I've loved Simplify for what feels like years, however today the Simplify v2 prompt came up forcing me to install v2. To my dismay, my beautiful, relaxing theme now hurts my eyes. Whereas in Simplify v1 the "white background" of the message pane appeared to have some transparency (allowing some of the background image to peek through), v2 seems to set the white background with no alpha, so it is 100% white, and my eyes are not happy about it. 

<img width="980" alt="simplify-message-background" src="https://user-images.githubusercontent.com/82321/107245772-868d9480-69fd-11eb-8ddd-f0f1179d8090.png">

Any idea what happened? Is the background transparency coming back any time soon?

**Comment by leggett** [comment]
Have you tried adjusting the theme to make the list dark?
<img width="844" alt="Screen Shot 2021-02-08 at 10 53 21 AM" src="https://user-images.githubusercontent.com/8763662/107267573-f13dd080-69fb-11eb-9b4f-bb4f4950afd4.png">

Currently, I don't have plans to bring back the transparency. I find it hurts readability. And since, you can change your threadlist to be dark instead of light I think it is a change for the better.

BUT, you are not the first to say something about this and I will continue to listen to feedback and reconsider if more people don't like the change. Closing for now but still listening.

**Comment by briantully** [comment]
Yes, I tried Dark, but it just makes the background Black. I thought the previous version with the slight transparency was the best choice and I didn't think it reduced readability at all, esp since the transparency was very low and opacity was high, (i.e., rgba(255,255,255,0.9). When using a beautiful/colorful background image that slight transparency makes a HUGE difference IMHO. Sorry to hear you don't have plans to bring it back :(

**Comment by leggett** [comment]
The readability is ok in some themes and really bad in others. It all depends on the background really and I don't have any way of detecting what it is for different backgrounds. 

While I don't have plans, I'm not closed to bringing it back... will revisit in the coming weeks.

**Comment by briantully** [comment]
Thanks. In the meantime there's always Stylebot Lite to the rescue ;)

**Comment by leggett** [comment]
I just looked at this and there is transparency. It is a very minimal 95% though:
`background-color: rgba(255, 255, 255, 0.95);`

I tried dropping that to 90% and my eyes are distracted... jumping between objects in the background that aren't scrolling and the content I am actually trying to read. 
https://user-images.githubusercontent.com/8763662/107321185-f419df00-6a56-11eb-9651-c2b9c044e5ea.mp4

I acknowledge that some themes are better with fewer objects in the background... but I believe that they are the exception with Gmail themes.

I'm still going to look at it again for reading pane but

**Comment by leggett** [comment]
Included in this issue is looking at the background for the list pane and nav with the reading pane enabled.
